homeassistant/components/unifi/sensor.py

```diff
@@ -272,6 +272,42 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda controller, obj_id: f"ac_power_conumption-{obj_id}",
         value_fn=lambda controller, device: device.outlet_ac_power_consumption,
     ),
+    UnifiSensorEntityDescription[Devices, Device](
+        key="Device uptime",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        has_entity_name=True,
+        allowed_fn=lambda controller, obj_id: True,
+        api_handler_fn=lambda api: api.devices,
+        available_fn=async_device_available_fn,
+        device_info_fn=async_device_device_info_fn,
+        event_is_on=None,
+        event_to_subscribe=None,
+        name_fn=lambda device: "Uptime",
+        object_fn=lambda api, obj_id: api.devices[obj_id],
+        should_poll=False,
+        supported_fn=lambda controller, obj_id: True,
+        unique_id_fn=lambda controller, obj_id: f"device_uptime-{obj_id}",
+        value_fn=lambda ctrlr, device: dt_util.now() - timedelta(seconds=device.uptime),
+    ),
+    UnifiSensorEntityDescription[Devices, Device](
```

Comment:
Doesn't this also need a native unit of measurement?

Reply:
There is none for the other uptime sensor. As it's a daytime object

Reply:
I think he means temperature which seems to be missing a UoM.

Reply:
In that case possibly

Reply:
Added native_unit_of_measurement